codalab/lib/cli_util.py

```diff
@@ -109,6 +111,13 @@ def get(dep):  # Return the key
                     'key %s exists with multiple values: %s and %s' % (key, key2val[key], val)
                 )
         else:
+            if key is None:
+                raise UsageError(
+                    'target_spec is empty. Please provide a valid target_spec in the format of {}.'.format(
+                        codalab.lib.bundle_cli.RUN_TARGET_SPEC_FORMAT
```

Comment:
This introduces a circular dependency since bundle_cli depends on cli_util.  Move the definitions to a new file and have both files include that.

Reply:
Can we move them to `cli_util.py` as there are already some formatting variable definied there? This way we can keep the orignial importing.

Reply:
Sure.

Reply:
Done.